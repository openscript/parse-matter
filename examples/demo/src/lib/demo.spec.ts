import { returnMarkdown } from './demo';

describe('demo', () => {
  it('should work', () => {
    expect(returnMarkdown()).toMatchSnapshot();
  });
});
