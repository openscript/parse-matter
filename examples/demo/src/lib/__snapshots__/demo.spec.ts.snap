// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`demo > should work 1`] = `
{
  "content": undefined,
  "data": undefined,
}
`;
